packages/runtime/container-runtime/src/gc/gcTelemetry.ts

```diff
@@ -352,15 +358,19 @@ export class GCTelemetryTracker {
 				return;
 			}
 			this.loggedUnreferencedEvents.add(uniqueEventId);
+			deletedNodeIds.push(nodeId);
+		}
+
+		if (deletedNodeIds.length > 0) {
 			logger.sendTelemetryEvent({
-				eventName: "GCObjectDeleted",
-				id: nodeId,
-				type: nodeType,
-				age: currentReferenceTimestampMs - nodeStateTracker.unreferencedTimestampMs,
-				timeout: this.configs.sweepTimeoutMs,
-				completedGCRuns,
-				lastSummaryTime,
-				...this.createContainerMetadata,
+				eventName: "GC_SweepReadyObjects_Delete",
+				details: JSON.stringify({
+					id: wrapWithCodeArtifact(JSON.stringify(deletedNodeIds)),
```

Comment:
Arrays are natively supported by the logger now!

Oh but anyway, you're further putting it in details...  The CodeArtifact tag will do nothing in there (e.g. it'll get passed through to Kusto).  You'd want to tag all of details as CodeArtifact, or probably just pull this detail out to its own thing.

Reply:
Don't do it.  I'm doing it in another PR for the whole codebase

Reply:
Are you saying I should not change anything here? Or still move id out?

Reply:
Sorry for the confusion.  I'm saying - Do move `id` out so it can be tagged, but don't try logging it directly as an array, still do `JSON.stringify` on it

Reply:
Fixed it.

Reply:
There were other instances too where a tagged property was part of details. Changed those too.